### ISSUE TITLE
unsuccessful gcs upload does not error out

### ISSUE THREAD

**Issue by matti** [sitespeedio/sitespeed.io#2769]
```
sitespeed.io_1   | [2019-11-08 08:07:38] ERROR: [sitespeedio.plugin.gcs] Could not upload to Google Cloud Storage Error: ENOENT: no such file or directory, open 'gcs-key.json''
sitespeed.io_1   | [2019-11-08 08:07:38] INFO: [sitespeedio] Find the result at ...
```

I think the process should exist with error != 0

**Comment by soulgalore** [comment]
Yep agree. We have concept where we pass important errors as a message on the queue, and they are picked at the end. If we have error the exit status is failure. Checked and we don't pass that on from gcs (or s3). It should be a couple of lines to fix it, but the errors right now is tied to a URL, that needs to be fixed (or let the upload plugins just use one of the tested URLs).

**Comment by soulgalore** [comment]
This was released in 11.1.0!
